docker-app/qfieldcloud/core/tests/test_api.py

```diff
@@ -36,9 +36,23 @@ def setUp(self):
 
     def test_api_status(self):
         response = self.client.get("/api/v1/status/")
+
         self.assertTrue(status.is_success(response.status_code))
-        self.assertEqual(response.json()["storage"], "ok")
-        self.assertEqual(response.json()["geodb"], "ok")
+
+        data = response.json()
+
+        self.assertIn("database", data)
+        self.assertEqual(data["database"], "ok")
+
+        self.assertIn("storages", data)
+        self.assertIsInstance(data["storages"], dict)
+
+        expected_storages = {"legacy_storage", "webdav", "default"}
```

Comment:
Can we actually get this values from the global `django...storages` value instead of hardcoding? I can imagine the `legacy_storage` to be gone quite soon.